data/859/017/47/85901747.geojson

```diff
@@ -46,7 +46,7 @@
         "Cringasi"
     ],
     "name:ron_x_preferred":[
-        "Cr\u00eenga\u0219i"
+        "Cr\u00e2nga\u0219i"
```

Comment:
Let's remove this record from the PR. Looks like a legitimate place; this PR will only track deprecated / not current neighbourhood changes.

Reply:
Ok, I'll have a separate PR for renaming some neighborhoods.